draft-lmbdhk-quic-multipath.md

```diff
@@ -132,6 +132,68 @@ Both identifiers have the same value, which is the sequence number of the connec
+manage the different paths. The first algorithm is the path manager (PM). Within the bounds
+set by the active_connection_id_limit transport parameter, the PM starts and stops paths based on
+the requirements of its application. The PM uses QUIC's path migration mechanism to create new paths and
+controls the termination of paths using the idle timeout and the exchange of PATH_ABANDON frames.
+It also monitors the utilization of the connection identifiers. We expect that different
+applications will use different paths managers based on their specific requirements. Since paths are
+always created by the client, a simple path manager would be to create one path per active network
+interface on the client. Another possibility would be to only create paths on the N first network
+interfaces of the client. A third possibility would be to only
+create additional paths if the existing path becomes congested.
+
+The second algorithm that multipath QUIC applications will require is a packet scheduler (PS).
+The PS is used when there are two or more paths that are active for a given multipath QUIC connection. In
+this case, QUIC must select the path over which each frame will be sent. Many PS strategies
+are possible and the best one will depend on the application's requirements. The PS should
+have access to (an abstraction of) the congestion control state of each path to be able to determine
+whether a path has enough capacity to carry a new packet. The PS also needs to have access to the
+path's MTU as different paths do not necessarily have the same MTU. A simple PS is a round-robin
+scheduler that checks the congestion window of a path before sending a QUIC packet.
+Another sample PS is an algorithm that selects the path whose congestion window is open and
+that has the smallest round-trip-time. This is the PS used by the Linux implementation of Multipath
+TCP. Note that a multipath QUIC PS will probably treat control and data packets differently. For
+example, a PS could favor low delay paths to send acknowledgments and higher bandwidth
+ones for data packets. A companion draft
+{{I-D.bonaventure-iccrg-schedulers}} provides several general-purpose multipath
+packet schedulers. 
+
+Multipath QUIC uses acknowledgments like regularly QUIC. More details about these acknowledgments
+are provided later. Multipath QUIC enables new techniques to cope with packet losses compared to QUIC.
+In QUIC, a lost packet can either be abandoned or retransmitted over the same path once it has been
+detected as lost. A Multipath QUIC implementation can use different strategies to cope with losses.
+A simple strategy is to retransmit a lost frame over the same path. Another strategy would be to
+simultaneously retransmit a lost frame over its original path and another one.
+A third strategy would be to use a dedicated path to send retransmissions. This document does not
+preclude a specific strategy that would eventually depend on the application needs.
+
```

Comment:
I agree with Mirja that this is a bit too detailed, probably too long. The path creation and packet scheduling functions are local functions defined by the implementation. We cannot really standardize what implementations do, so the best is to just have a mention that these functions exist, and leave it at that.